app/controllers/v1/terms_controller.rb

```diff
@@ -0,0 +1,123 @@
+module V1
+  class TermsController < ApplicationController
+    # Check that vocabulary is valid, if not should render and error
```

Comment:
Is this a TODO note?

Reply:
Yea, this is a TODO note. I can add `TODO:` in front of it if that's helpful.

Reply:
Thanks! I'm in the habit of periodically searching apps for the string "TODO" to find all TODOs, so that would be great.